chasm/src/testData/results/other/test_merge_insns.result

```diff
@@ -0,0 +1,91 @@
+// class version 61.0 (61)
+// access flags 0x21
+public class other/TestMergeInsns {
+
+  // compiled from: TestMergeInsns.java
+  // access flags 0x19
+  public final static INNERCLASS java/lang/invoke/MethodHandles$Lookup java/lang/invoke/MethodHandles Lookup
+
+  // access flags 0x1
+  public <init>()V
+   L0
+    LINENUMBER 3 L0
+    ALOAD 0
+    INVOKESPECIAL java/lang/Object.<init> ()V
+    RETURN
+   L1
+    LOCALVARIABLE this Lother/TestMergeInsns; L0 L1 0
+    MAXSTACK = 1
+    MAXLOCALS = 1
+
+  // access flags 0x9
+  public static source()V
+   L0
+    LINENUMBER 5 L0
+    ICONST_0
+    ISTORE 0
+   L1
+   FRAME APPEND [I]
+    ILOAD 0
+    ICONST_5
+    IF_ICMPGE L2
+   L3
+    LINENUMBER 6 L3
+    GETSTATIC java/lang/System.out : Ljava/io/PrintStream;
+    ILOAD 0
+    INVOKEDYNAMIC makeConcatWithConstants(I)Ljava/lang/String; [
+      // handle kind 0x6 : INVOKESTATIC
+      java/lang/invoke/StringConcatFactory.makeConcatWithConstants(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/String;[Ljava/lang/Object;)Ljava/lang/invoke/CallSite;
+      // arguments:
+      "Test Source Method x\u0001"
+    ]
+    INVOKEVIRTUAL java/io/PrintStream.println (Ljava/lang/String;)V
+   L4
+    LINENUMBER 5 L4
+    IINC 0 1
+    GOTO L1
+   L2
+    LINENUMBER 8 L2
+   FRAME SAME
+    RETURN
+    LOCALVARIABLE i I L1 L2 0
+    MAXSTACK = 2
+    MAXLOCALS = 1
+
+  // access flags 0x9
+  public static target()V
+    LDC "Dummy Condition"
+    INVOKESTATIC java/lang/System.getProperty (Ljava/lang/String;)Ljava/lang/String;
+    IFNONNULL L0
+   L1
+   FRAME SAME
+    GETSTATIC java/lang/System.out : Ljava/io/PrintStream;
+    LDC "Test Target Method"
+    INVOKEVIRTUAL java/io/PrintStream.println (Ljava/lang/String;)V
+   L2
+    LINENUMBER 11 L2
+    ICONST_0
+    ISTORE 0
+   L0
+    LINENUMBER 14 L0
+   FRAME SAME
+    ILOAD 0
+    ICONST_5
+    IF_ICMPGE L1
```

Comment:
This jump is wrong. It's targeting the label in the source method.
Not sure if we want to fix that, or how we would even do that, but I think it's worth pointing out.
This is out of scope of this PR though.